site/src/pages/docs/advanced-streaming-backpressure.mdx

```diff
@@ -1,27 +1,22 @@
-# Sending a streaming response with backpressure
+# Sending a streaming response with back pressure
 
-Let's say that we want to serve a static file whose size is larger than your available memory.
-If we load the file into the memory at once, we will definitely get an `OutOfMemoryError`.
-In order to avoid the error, we should:
-1. Divide the file into chunks
-2. Load the first chunk into memory
-3. Send the chunk to the client
-4. Wait until the chunk is written to the sending socket buffer
-   - So we don't load data into memory when the client is not ready to receive. This is called __backpressure__.
-5. Load the second chunk and repeat the steps until we send all chunks
+Suppose we want to serve a static file whose size is larger than your available memory. If we load the file into the memory at once, we will definitely get an `OutOfMemoryError`.
 
-Now, we can send the large file without spending much memory.
+To prevent such error and send large files without consuming too much memory:
 
-<Tip>
+1. Divide the file into chunks.
+2. Load the first chunk into memory.
+3. Send the chunk to the client.
+4. Wait until the chunk is written to the sending socket buffer. (See below for explanation.)
+5. Load the second chunk and repeat the steps 3 and 4 until we send all chunks.
 
-See [Let’s Play with Reactive Streams on Armeria - 1](https://engineering.linecorp.com/en/blog/reactive-streams-armeria-1/)
-to understand backpressure and the situation when an `OutOfMemoryError` is raised.
+Waiting for the chunk to be written is to avoid loading into memory when the client is not ready to receive. This is called __back pressure__. 
+See [Let’s Play with Reactive Streams on Armeria - 1](https://engineering.linecorp.com/en/blog/reactive-streams-armeria-1/) to learn back pressure and what happens when an `OutOfMemoryError` is raised.
```

Comment:
How about 
```suggestion
See [Let’s Play with Reactive Streams on Armeria - 1](https://engineering.linecorp.com/en/blog/reactive-streams-armeria-1/) to learn back pressure and when an `OutOfMemoryError` is raised.
```
Because I wanted to say that an `OutOfMemoryError` is raised if we don't use back pressure.
We all know what happens when an `OutOfMemoryError` is raised. 😄

Reply:
@minwoox , 
Hope this fix is as you expected
https://github.com/line/armeria/pull/3847/commits/1ad1e22b7fd50f4cece2d1a566ce2c53d35941bb